gadopt/momentum_equation.py

```diff
@@ -18,26 +18,21 @@
 
 and then return `-F`.
 
-Users should not interact with these classes; instead, please use
-the solver provided in the stokes_integrators module.
-
 """
 
-from typing import Optional
-
-import firedrake as fd
-from firedrake import dot, inner, outer, transpose, nabla_grad, div
-from firedrake import avg, Identity, jump
-from firedrake import FacetArea, CellVolume
+from firedrake import *
```

Comment:
I know this has come up in meetings before... How come you are now importing all of firedrake?

Reply:
Putting this link [here](https://stackoverflow.com/questions/710551/use-import-module-or-from-module-import), as it can be useful.
I decided to use `from firedrake import *` because we now have a lot of objects in this module that come from `firedrake` and other developers have argued against `import firedrake as fd` in the past. Listing all the objects imported seems tedious to me and results in a never-ending import statement. From a Python perspective, I still think the right thing to do is `import firedrake as fd`, but I also see why it is not popular among developers. @angus-g @dsroberts Any opinions here?

Reply:
thanks for the link

Reply:
I think `from firedrake import *` is better than the list of explicit imports. Yes, it would be nice to be consistent across all the g-adopt modules, but that's not in the scope of this PR. Firedrake/UFL is _almost_ its own DSL, so I don't think the usual rules apply anyway.

Reply:
For those who do not know what a DSL is: https://www.jetbrains.com/mps/concepts/domain-specific-languages/